### PR TITLE
Refactor usage

### PR DESCRIPTION
**Pull request by shimonewman**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "app/console"]
 	path = app/console
 	url = https://github.com/appwrite/console
-	branch = 3.2.3
+	branch = feat-usage-sh-1.4
```
Comment (christyjacob4) [inline .gitmodules:4]: lets remember to keep the frontend team in the loop about this 👍

File: app/config/variables.php [hunk 2]
```diff
@@ -335,54 +335,6 @@
...
-                'introduction' => '',
-                'default' => '8086',
-                'required' => false,
-                'question' => '',
-                'filter' => ''
-            ],
-        ],
-    ],
-    [
-        'category' => 'StatsD',
```
Comment (christyjacob4) [inline app/config/variables.php:363]: lets not remove the variables instead mark them as `deprecated` 👍

File: app/controllers/api/databases.php [hunk 3]
```diff
@@ -3645,107 +3580,57 @@ function updateAttribute(
...
-            ];
+        Authorization::skip(function () use ($dbForProject, $days, $metrics, &$stats) {
+            foreach ($metrics as $metric) {
+                $limit = $days['limit'];
+                $period = $days['period'];
+                $results = $dbForProject->find('stats', [
+                    Query::equal('period', [$period]),
+                    Query::equal('metric', [$metric]),
+                    Query::limit($limit),
+                    Query::orderDesc('time'),
```
Comment (christyjacob4) [inline app/controllers/api/databases.php:3600]: do we have appropriate indexes for these ?

File: docker-compose.yml [hunk 4]
```diff
@@ -648,19 +644,21 @@ services:
...
     image: appwrite-dev
     networks:
       - appwrite
     volumes:
       - ./app:/usr/src/code/app
       - ./src:/usr/src/code/src
-      - ./dev:/usr/local/dev
+      - ./vendor/utopia-php/platform:/usr/src/code/vendor/utopia-php/platform
+      - ./vendor/utopia-php/framework:/usr/src/code/vendor/utopia-php/framework
+      - ./vendor/utopia-php/queue:/usr/src/code/vendor/utopia-php/queue
```
Comment (christyjacob4) [inline docker-compose.yml:659]: lets remove these

File: docker-compose.yml [hunk 5]
```diff
@@ -843,26 +838,6 @@ services:
...
-
-  telegraf:
-    image: appwrite/telegraf:1.4.0
-    container_name: appwrite-telegraf
-    <<: *x-logging
-    networks:
-      - appwrite
-    environment:
-      - _APP_INFLUXDB_HOST
-      - _APP_INFLUXDB_PORT
```
Comment (christyjacob4) [inline docker-compose.yml:864]: we also need to remove these from the compose.phtml file

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**